src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/pureThermalDiffusion/Example.rst

```diff
@@ -0,0 +1,105 @@
+.. _AdvancedExamplePureThermalDiffusionWellbore:
+
+
+####################################################
+Pure thermal diffusion around a wellbore
+####################################################
+
+------------------------------------------------------------------
+Problem description
+------------------------------------------------------------------
+
+This example uses the thermal flow solver to a pure thermal diffusion problem around a wellbore. Thermal convection and fluid flow are removed by settting fluid pressure and fluid heat capacity to zero to mimic a pure thermal diffusion problem, .
+
+Analytical results of the temperature profile along the radial directionis `(Wang and Papamichos, 1994) <https://agupubs.onlinelibrary.wiley.com/doi/abs/10.1029/94WR01774>`__ :
+
+.. math::
+   T(r) = T_{in}\sqrt{\frac{R_{in}}{r}}erfc(\frac{r-R_{in}}{2\sqrt{c_{T}t}})
+
+where :math:`r` is the radial coordinate, :math:`T_{in}` the temperature applied on the surface of the wellbore at :math:`r = R_{in}`, :math:`c_{T}` the thermal diffusion coefficient of rock that is the ratio between the thermal conductivity and ther volumetric heat capacity of rock.
+
+
+**Input file**
+
+This benchmark example uses no external input files and everything required is
+contained within two GEOSX xml files that are located at:
+
+.. code-block:: console
+
+  inputFiles/singlePhaseFlow/thermalCompressible_2d_base.xml
+
+and
+
+.. code-block:: console
+
+  inputFiles/singlePhaseFlow/thermalCompressible_2d_benchmark.xml
+
+The corresponding integrated test is
```

Comment:
Smoke test is not found in this PR and would be added into integratedTests separately?

Reply:
It is already added before this PR. The xml file defining the smoke test is inputFiles/singlePhaseFlow/thermalCompressible_2d_smoke.xml